demos/CPP_waitForTriggerDemo.m

```diff
@@ -1,6 +1,6 @@
 cd ..;
 
-cfg.device = 'Scanner';
+cfg.device = 'mri';
```

Comment:
```suggestion
cfg.testingDevice = 'mri';
```

Reply:
should be!  
but the demo was crashing for another reason, cfg.MRI is not a defined field. And yet, it was called with isempty(cfg.MRI.xxx)